### PR TITLE
Version number in shop admin

### PR DESCRIPTION
**Pull request by shawnadelic**
Added the Shoop version number to the admin dashboard to help merchant/
admin troubleshooting process

screenshot from 2016-01-14 09 35 28

Refs SHOOP-1999

### CODE REVIEW

**Comment by suutari-ai** [comment]
test this please

File: shoop/admin/views/dashboard.py [hunk 1]
```diff
@@ -17,6 +18,7 @@ class DashboardView(TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super(DashboardView, self).get_context_data(**kwargs)
+        context["version"] = _version.__version__
```
Comment (suutari-ai) [inline shoop/admin/views/dashboard.py]: You shouldn't import or use a private submodule from another python package, i.e. one that starts with an underscore (in this case `_version`). Please just `import shoop` and use `shoop.__version__`.

File: shoop/admin/templates/shoop/admin/dashboard/dashboard.jinja [hunk 2]
```diff
@@ -96,6 +96,9 @@
                 {% if activity %}
                     {% call regular_block("medium") %}{{ activity_block_content(activity) }}{% endcall %}
                 {% endif %}
+                <div class="block width-large">
+                    <p>{% trans %}Shoop version{% endtrans %}: {{ version }}</p>
```
Comment (suutari-ai) [inline shoop/admin/templates/shoop/admin/dashboard/dashboard.jinja]: I don't think this is very translator friendly, since for some language it is possible that version number has to be before the text or between some words. Also, IMHO the word version is not needed in this context, it should be easy enough to understand that "3.0" in "Shoop 3.0" is a version number.

TLDR: Use:
```
<p>{{ _("Shoop %(version)s", version=version) }}</p>
```

**Comment by shawnadelic** [comment]
test this please

**Comment by suutari-ai** [comment]
Good work, Shawn! Thanks!

**Comment by akx** [comment]
Next time though, please use imperative case and present tense in the commit message:

> Add version number to shop admin

etc. :)
